Test/civl/large-samples/treiber-stack.bpl

```diff
@@ -6,39 +6,44 @@ function {:inline} AllLocPieces(): Set LocPiece {
   Set_Add(Set_Add(Set_Empty(), Left()), Right())
 }
 
+type LocTreiberStack = KeyedLoc LocPiece;
 type LocTreiberNode = KeyedLoc LocPiece;
 type StackElem T = Node LocTreiberNode T;
 type StackMap T = Map LocTreiberNode (StackElem T);
 datatype Treiber<T> { Treiber(top: Option LocTreiberNode, {:linear} nodes: StackMap T) }
 
+function {:inline} LeftLocPiece(loc_piece: KeyedLoc LocPiece): KeyedLoc LocPiece {
```

Comment:
We're not using `KeyedLoc LocPiece` directly anywhere else. Should we use a type synonym and rename the function appropriately?

Reply:
Yes, good idea. I will make that change.